gulpfile.js

```diff
@@ -60,7 +60,8 @@ var gulp = require('gulp'),
             singleRun: true
         },
         sass: {
-            includePaths: bourbon.includePaths
+            includePaths: bourbon.includePaths,
+            sourceComments: true
```

Comment:
Yes, for now. According to Pete, this won't affect performance and it's very useful to me for debugging.